### PR TITLE
Adds Life After Nil

### PR DESCRIPTION
**Pull request by samjonester**

### CODE REVIEW

**Comment by samjonester** [comment]
Will proof the PR url and merge in the morning...

File: app/posts/2017-01-18-life-after-nil.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  url: "https://player.vimeo.com/video/200077718"
+  type: "vimeo"
+tldr: 
+  body: """
+        Ruby is late to the type system party. Let’s give Ruby something smart to say when it gets there.
+        """
+reddit: false
+---
+
+There are murmurs that a type system will be coming to Ruby. Before that happens, you should get informed about what is hot in the current type system market. Haskell is known for it’s type system, but instead of describing it with dense language, let’s take a journey through code examples. Throughout our trip, we will avoid scary buzzwords like “monad” and “algebraic data type” because, honestly, what good is a formal definition when you don’t understand the power behind the concept.
```
Comment (jasonkarns) [inline app/posts/2017-01-18-life-after-nil.md]: {it's -> its} type system

Comment (jasonkarns) [inline app/posts/2017-01-18-life-after-nil.md]: Last sentence in paragraph is a question. Needs question mark.

File: app/posts/2017-01-18-life-after-nil.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+We will compare solutions in Ruby to solutions in Haskell, and each stop on our trip will introduce a new mind blowing paradigm brought to you by Haskell’s type system. You will learn about type systems through a non-threatening story, and you’ll understand the value a type system can bring to your code. Key stops on our route will be “forget about nil”, “declaratively model your domain”, and “allow your compiler to drive your design”.
```
Comment (jasonkarns) [inline app/posts/2017-01-18-life-after-nil.md]: introduce a new, mind-blowing paradigm

**Comment by samjonester** [comment]
Thanks, Jason!

**Pull request merged**